tools/clang/test/CodeGenSPIRV/vk.combined-image-sampler.binding-shift.hlsl

```diff
@@ -0,0 +1,14 @@
+// RUN: %dxc -T ps_6_0 -E main -fvk-t-shift 10 0 -fvk-s-shift 20 0
+
+// CHECK: OpDecorate %tex DescriptorSet 0
+// CHECK: OpDecorate %tex Binding 10
+[[vk::combinedImageSampler]]
+SamplerState sam : register(s0);
+[[vk::combinedImageSampler]]
+Texture2D<float4> tex : register(t0);
+
```

Comment:
Could you please add `SamplerState sam0 : register(s0);` and `SamplerState sam1 : register(s1);` and check
```
// CHECK: OpDecorate %sam0 Binding 20
// CHECK: OpDecorate %sam1 Binding 21
```
?

Reply:
Done